src/libcsg/modules/io/h5mdtrajectoryreader.cc

```diff
@@ -22,6 +22,7 @@
 
 // Third party includes
 #include <hdf5.h>
+#include <boost/regex.hpp>
```

Comment:
I think std:: has a regex as well now? I am not sure if we still use it, because of some distribution problem @junghans

Reply:
I don't think boost-regex is big burden.